var/spack/repos/builtin/packages/py-multiecho/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMultiecho(PythonPackage):
+    """Combine multi-echoes from a multi-echo fMRI acquisition."""
+
+    homepage = "https://github.com/Donders-Institute/multiecho"
+    pypi = "multiecho/multiecho-0.28.tar.gz"
+
+    version("0.28", sha256="d0459bd03398547116d8e989b2d2b7922af0ae7ae77e233794dd7253a2abced3")
+
+    depends_on("python@3.6:3.9", type=("build", "run"))
```

Comment:
I don't see any `python_requires` so the supported versions of Python should be unrestricted, I would remove this line.

Reply:
The python restriction I got from here:
https://github.com/Donders-Institute/multiecho/blob/0.28/pyproject.toml#L11

Reply:
Hmm, I'm not actually sure if setuptools reads that part of the file, but let's keep it for now.